snuba/consumers/consumer_config.py

```diff
@@ -129,6 +131,8 @@ def _resolve_env_config() -> EnvConfig:
     lower_retention_days = settings.LOWER_RETENTION_DAYS
     valid_retention_days = list(settings.VALID_RETENTION_DAYS)
     record_cogs = settings.RECORD_COGS
+    clickhouse_user = str(settings.CLUSTERS[0].get("user"))
```

Comment:
will we always have the same user and password for all clusters? can we use the `StorageConfig` instead of the `EnvConfig`?